mount.cc

```diff
@@ -156,11 +170,19 @@ NAN_METHOD(MountSync) {
   std::string s_fsType(*fsType);
   std::string s_dataStr(*dataStr);
 
-  int ret = mount(s_devFile.c_str(),
-          s_target.c_str(),
-          s_fsType.c_str(),
-          options->Value(),
-          s_dataStr.c_str());
+  #ifdef __linux__
+    int ret = mount(s_devFile.c_str(),
+            s_target.c_str(),
+            s_fsType.c_str(),
+            options->Value(),
+            s_dataStr.c_str());
+  #elif __APPLE__
```

Comment:
I don't think so. I'd have to use `#if defined(__FreeBSD__) || defined(__APPLE__)`